umm-spec-lib/src/cmr/umm_spec/migration/version_migration.clj

```diff
@@ -221,6 +221,19 @@
      (dissoc :ComposedOf)
      (dissoc :NumberOfInstruments)))
 
+(defn migrate-tiling-identification-systems
+  "Migrate TilingIdentificationSystems from 1.9 up to 1.10"
```

Comment:
Comment on what the function is doing - it's dropping those that don't translate. @eereiter is that correct functionality?